Cql/Cql.Compiler/ExpressionBuilder.cs

```diff
@@ -2035,26 +2373,59 @@ protected Expression FunctionRef(FunctionRef op, ExpressionBuilderContext ctx)
 
         protected Type GetFuncType(Type[] funcTypeParameters)
         {
-            Type? funcType = funcTypeParameters.Length switch
-            {
-                0 => throw new NotSupportedException(),
-                1 => typeof(Func<>).MakeGenericType(funcTypeParameters),
-                2 => typeof(Func<,>).MakeGenericType(funcTypeParameters),
-                3 => typeof(Func<,,>).MakeGenericType(funcTypeParameters),
-                4 => typeof(Func<,,,>).MakeGenericType(funcTypeParameters),
-                5 => typeof(Func<,,,,>).MakeGenericType(funcTypeParameters),
-                6 => typeof(Func<,,,,,>).MakeGenericType(funcTypeParameters),
-                7 => typeof(Func<,,,,,,>).MakeGenericType(funcTypeParameters),
-                8 => typeof(Func<,,,,,,,>).MakeGenericType(funcTypeParameters),
-                9 => typeof(Func<,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                10 => typeof(Func<,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                11 => typeof(Func<,,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                12 => typeof(Func<,,,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                13 => typeof(Func<,,,,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                14 => typeof(Func<,,,,,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                15 => typeof(Func<,,,,,,,,,,,,,,>).MakeGenericType(funcTypeParameters),
-                _ => throw new NotSupportedException("Fucntions with more than 15 parameters are not supported."),
-            };
+            Type? funcType;

```

Comment:
I pulled in the changes from the `develop` branch with this PR as well, since we were many commits behind.  This must have been made to `develop` some time after we branched the main `cql-to-elm` branch.